src/shared/components/plots/WaterfallPlot.tsx

```diff
@@ -60,18 +60,27 @@ const DEFAULT_FONT_FAMILY = "Verdana,Arial,sans-serif";
 export const LEGEND_Y = 30
 const RIGHT_PADDING = 120; // room for correlation info and legend
 const NUM_AXIS_TICKS = 8;
-const PLOT_DATA_PADDING_PIXELS = {y: 3, x: [30, 30]};
+const PLOT_DATA_PADDING_PIXELS = (horizontal:Boolean) => {
+    if (horizontal) {
+        return 30;
+    }
+    return {y: 3, x: 30};
+}
```

Comment:
This is not a constant. No captalization should be used here. This should be refactored into a @computed field `dataPaddingInPixels` as:
```
@computed get dataPaddingInPixels() {
   if (this.props.horizontal) {
        return 30;
    }
    return {y: 3, x: 30};
}
```

Also, this method returns different data types depending on the value of `horizontal`. Is this supported by TypeScript? I guess you have to add a return type and refactor to have the code handle it.